File: src/main/java/com/marknkamau/Exceptions/FileDoesNotExist.java
package com.marknkamau.Exceptions;

public class FileDoesNotExist extends BaseException {
    private static final String TITLE = "File does not exist";
    private static final String DOES_NOT_EXIST = "Input file does not exist.";

    public FileDoesNotExist() {
        super(TITLE, DOES_NOT_EXIST);
    }
}

File: src/main/java/com/marknkamau/Exceptions/FileDoesNotExist.kt
package com.marknkamau.Exceptions

class FileDoesNotExist : BaseException(FileDoesNotExist.TITLE, FileDoesNotExist.DOES_NOT_EXIST) {
    companion object {
        private val TITLE = "File does not exist"
        private val DOES_NOT_EXIST = "Input file does not exist."
    }
}
